### ISSUE TITLE
403: Can't pull from private v2 registry

### ISSUE THREAD

**Issue by dj-hedgehog** [rancher/rancher#5198]
**Rancher Version:** v1.1.0-dev1
**Docker Version:** 1.11.2
**OS and where are the hosts located? (cloud, bare metal, etc):** bare metal
**Setup Details: (single node rancher vs. HA rancher, internal DB vs. external DB)** sindle node rancher, external DB
**Environment Type: (Cattle/Kubernetes/Swarm/Mesos)** Cattle

I'm using GitLab as a private container-registry. I set up this private registry in `Infrastructure > Registries` with the correct authentication data but upon hitting `rancher-compose up` the container which has to be pulled from my private registry is created over and over again with this error:

> Error (500 Server Error: Internal Server Error ("Get https://registry.example.com/v2/server/backend/manifests/latest: error parsing HTTP 403 response body: unexpected end of JSON input: """))

Do you have an idea what might cause this problem? Using `docker-compose up` with the same Dockerfile works (after using `docker login` of course).

**Comment by deniseschannon** [comment]
Have you tried using the same image and using the UI to start a service? 

What is your docker-compose file that you are using?

**Comment by dj-hedgehog** [comment]
Hey @deniseschannon 

I just started it from the UI and am getting `Error (Image [registry.example.com/myproject/backend:latest] failed to pull: Error: image myproject/backend:latest not found)`

The image is there and can be pulled via `docker pull` without any issues. I even `docker login` to the registry on the host executing the image.

If it helps I could create an account in my registry for you and push a test-image. 

This is the `docker-compose.yml`:

```
backend:
  image: registry.example.com/myproject/backend:latest
  links:
    - db
  ports:
    - 1337:1337/tcp
  environment:
    - NODE_ENV=production
  labels:
    io.rancher.container.pull_image: always
    io.rancher.scheduler.affinity:host_label: system=prod
    VIRTUAL_HOST: backend.test.example.com

db:
  image: postgres:latest
  environment:
    - POSTGRES_USER=foo
    - POSTGRES_PASSWORD=bar
```

**Comment by sangeethah** [comment]
@dj-hedgehog: For the docker-compose.yml you have provided , registry entry added in rancher-server should have "Address" field as "registry.example.com" (without http:// or https://). Can you check this setting?

**Comment by dj-hedgehog** [comment]
@sangeethah By trying to make a screenshot for you (and changing the registry URL via chrome's HTML inspector) I found out that I had a space in my username which was wrong.
I started the `docker-compose.yml` above again.

Now the container from the private registry is starting up and instantly being stopped again without me having the chance to look into its logs. How can I view what's going wrong this time?
